code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -116,6 +116,20 @@
 	if(hotness && reagents)
 		reagents.expose_temperature(hotness)
 		to_chat(user, "<span class='notice'>You heat [name] with [I]!</span>")
+
+	//Cooling method
+	if(istype(I, /obj/item/extinguisher))
+		var/obj/item/extinguisher/extin = I
+		if(extin.safety)
+			return
+		if (extin.reagents.total_volume < 1)
+			to_chat(user, "<span class='warning'>\The [extin] is empty!</span>")
+			return
+		var/cooling = (0 - reagents.chem_temp)*(extin.cooling_power*2)
+		reagents.expose_temperature(cooling)
+		to_chat(user, "<span class='notice'>You cool the [name] with the [I]!</span>")
+		playsound(src.loc, 'sound/effects/extinguish.ogg', 75, TRUE, -3)
+		extin.reagents.remove_all(1)
```

Comment:
```suggestion
		var/obj/item/extinguisher/extinguisher = I
		if(extinguisher.safety)
			return
		if(extinguisher.reagents.total_volume < 1)
			to_chat(user, "<span class='warning'>\The [extinguisher] is empty!</span>")
			return
		var/cooling = (0 - reagents.chem_temp) * (extinguisher.cooling_power * 2)
		reagents.expose_temperature(cooling)
		to_chat(user, "<span class='notice'>You cool the [name] with the [I]!</span>")
		playsound(loc, 'sound/effects/extinguish.ogg', 75, TRUE, -3)
		extinguisher.reagents.remove_all(1)
```
* Make your vars readable.
* src is implied.
* use `if()` or `if ()`, whatever you prefer, but try to be consistent